submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,146 @@
+
+Now that we've seen how nuclear power is not so dangerous in the vanilla game, let's look over the other common concerns about it.
+
+### **Common Concerns with Nuclear Power**
+
+* **Nuclear designs and calculations are very complex** – There is nothing wrong about starting with small and simple reactors, and the rest boils down to a few rules of thumb and some ratios. In the section "Designing your own Nuclear Plants" I hope to cover the details and point to guides and tools you can use.
+
+* **Nuclear power is too expensive to set up until the late game** – Nuclear power is unlocked with chemical science packs. While it does take some time to get everything running, nuclear power actually costs less in terms of raw materials when compared to equivalent amounts of solar power, especially when the neighbor bonus comes into play. I made an analysis about this and [posted it on Reddit](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/). I found that even with a basic 40MW nuclear setup, you save thousands of copper and iron plates when compared to building 40MW of solar panels and accumulators. Hence if your power demand passes 40MW and you don't want boiler pollution, you may find it quite profitable to get into nuclear power.
+
+* **Nuclear power is a big polluter** - In vanilla Factorio, nuclear radiation causes no damage and has no effects on biter evolution, while regular pollution does. The pollution caused by nuclear power comes only from the mining and processing of uranium ore. I compared this to the pollution from coal power in the same [Reddit post](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) as above. I found that nuclear power causes only 2-10% of the pollution of mining and burning coal for power.
+
+* **Nuclear reactors are far too expensive to maintain without the Kovarex Enrichment Process** – Obtaining one piece of Uranium-235 indeed requires centrifuging over 1000 of uranium ore pieces, but on the other hand a single piece of it is enough to make 10 uranium fuel cells, which last over 33 minutes in total. In a rather eye-opening [tutorial video](https://youtu.be/Qw_NzPuccxk?t=1610), Nilaus walks us through the mathematics and from there we see that you need only eight electric miners and one centrifuge to maintain one nuclear reactor continuously without the Kovarex Enrichment Process, although it does not hurt to process a little more uranium in preparation for unlocking Kovarex or in case of bad luck during centrifuging. In addition, placing any productivity modules you can get in the centrifuges and fuel assemblers improves your yield significantly.
```

Comment:
```suggestion
* **Nuclear reactors are far too expensive to maintain without the Kovarex Enrichment Process** – Obtaining one piece of Uranium-235 indeed requires centrifuging over 1000 uranium ore pieces, but on the other hand a single piece of it is enough to make 10 uranium fuel cells, which last over 33 minutes in total. In a rather eye-opening [tutorial video](https://youtu.be/Qw_NzPuccxk?t=1610), Nilaus walks us through the mathematics and from there we see that you need only eight electric miners and one centrifuge to maintain one nuclear reactor continuously without the Kovarex Enrichment Process, although it does not hurt to process a little more uranium in preparation for unlocking Kovarex or in case of bad luck during centrifuging. In addition, placing any productivity modules you can get in the centrifuges and fuel assemblers improves your yield significantly.
```